starters/gatsby-starter-blog/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "gatsby-starter-blog",
+  "description": "Starter Gatsby Blog",
+  "version": "1.0.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
+  "bugs": {
+    "url": "https://github.com/gatsbyjs/gatsby-starter-blog/issues"
+  },
+  "dependencies": {
+    "gatsby": "^2.0.19",
```

Comment:
Not really sure of what our approach is for keeping these up to date, but we'll need to make sure it still works with the monorepo approach.